### PR TITLE
Update mirrors and deps

### PR DESCRIPTION
**Pull request by Swader**
Just an update.

### CODE REVIEW

File: docs/mirror-learn-governance.md [hunk 1]
```diff
@@ -142,62 +185,111 @@ electorate = 1500
...
+| 16  | 5   |
+| 32  | 6   |
 
+Based on the genesis runtime, the maximum number of lock periods is set to 6 and the lock period is
+30 days on Polkadot and eight days on Kusama.
 
 #### Adaptive Quorum Biasing
 
-Polkadot introduces a concept "Adaptive Quorum Biasing", which functions as a lever that the council can use to alter the effective super-majority required to make it easier or more difficult for a proposal to pass in the case that there is no clear majority of voting power backing it or against it.
+Polkadot introduces a concept "Adaptive Quorum Biasing", which functions as a lever that the
```
Comment (laboon) [inline docs/mirror-learn-governance.md:220]:
```suggestion
Polkadot introduces the concept of "Adaptive Quorum Biasing", which functions as a lever that the
```

File: docs/mirror-learn-governance.md [hunk 2]
```diff
@@ -142,62 +185,111 @@ electorate = 1500
...
+against - equate to a simple majority carries system at 100% turnout.
 
 ## Council
 
-To represent passive stakeholders, we introduce the idea of a "council". The council is an on-chain entity comprising a number of actors each represented as an on-chain account. For Polkadot this number is likely to begin at around six people, and increase over the course of 9 months to 24 people (roughly one extra individual coming on every two weeks). In general it has a fixed number of seats (envisioned to be 24 for Polkadot) and all members have a fixed term (12 months).
+To represent passive stakeholders, we introduce the idea of a "council". The council is an on-chain
+entity comprising a number of actors, each represented as an on-chain account. For Polkadot this
+number is likely to begin at around six people, and increase over the course of 9 months to 24
+people (roughly one extra individual coming on every two weeks). In general it has a fixed number
+of seats (envisioned to be 24 for Polkadot) and all members have a fixed term (12 months).
```
Comment (laboon) [inline docs/mirror-learn-governance.md:256]:
```suggestion
of seats and all members have a fixed term (12 months).
```

This is redundant with the sentence before it.

File: docs/mirror-maintain-guides-how-to-nominate-kusama.md [hunk 3]
```diff
@@ -32,7 +32,9 @@ Click on "Nominate" on an account you've bonded and you will be presented with a
 
 ![Nominating validators](/img/NPoS/nominate.png)
 
-Select them, confirm the transaction, and you're done - you are now nominating. You should notice your balance increasing shortly.
+Select them, confirm the transaction, and you're done - you are now nominating.  Your nominations will become active in the next era.  Eras last four hours on Kusama - depending on when you do this, your nominations may become active almost immediately, or you may have to wait almost the entire four hours before your nominations are active.  You can chek how far along Kusama is in the current era on the [Staking page](https://polkadot.js.org/apps/#/staking).
```
Comment (laboon) [inline docs/mirror-maintain-guides-how-to-nominate-kusama.md:35]:
```suggestion
Select them, confirm the transaction, and you're done - you are now nominating.  Your nominations will become active in the next era.  Eras last six hours on Kusama - depending on when you do this, your nominations may become active almost immediately, or you may have to wait almost the entire six hours before your nominations are active.  You can check how far along Kusama is in the current era on the [Staking page](https://polkadot.js.org/apps/#/staking).
```

**Review by laboon — CHANGES_REQUESTED** [review]
Looks like you caught some of the stuff from my earlier PR in here, rebasing should make it go away

**Comment by Swader** [comment]
Should be good now

**Comment by lsaether** [comment]
Do we need to update mirrors with PRs or does it do in the build process now?

**Comment by Swader** [comment]
It is not in the build process yet, for now it's still manual. Optimally, the author of a change would run mirror.js before committing.

**Comment by lsaether** [comment]
Ok we can track that here #366

**Comment by lsaether** [comment]
It still looks like Bill's comments weren't addressed, at least I don't see "outdated" in the header so I assume this PR still built the old versions?

**Comment by lsaether** [comment]
outdated after #389

**Pull request closed without merging**